### PR TITLE
Use libService for ipv6 tracing.

### PR DESCRIPTION
**Pull request by tommikas**
Also fixed the mbed_trace_ipv6_prefix function and related unittests.

### CODE REVIEW

**Comment by tommikas** [comment]
@jupe @kjbracey-arm

**Comment by jupe** [comment]
lgtm

**Comment by jupe** [comment]
@pan- could you check that this not cause any problems for ble?

File: test/stubs/common_functions_stub.c [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2014-2015 ARM Limited. All rights reserved.
```
Comment (korjaa) [inline test/stubs/common_functions_stub.c]: 2016 :)

Comment (tommikas) [inline test/stubs/common_functions_stub.c]: Oh whoops.. I doubt that'd affect the legality, but still, should change that.

**Pull request merged**